HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,736 @@
+| 0x… | <htsTokenAddress>.rejectTokenFT() | Y |
+| 0x… | <htsTokenAddress>.rejectTokensNFT(int64[] serialNumber) | Y |
+| 0x… | <htsSystemContractAddress>.rejectTokens(address[] memory ftAddresses, NftId[] memory nftIds) |  |
+| 0x… | <hasSystemContractAddress>.setAutomaticAssociations(boolean enableAutoAssociations) | Y |
+
+To increase API reuse and for consistency the input objects will closely match the HAPI protobuf objects.
+
+### SDK
+
+The SDKs must support the new HAPI transactions
+
+- `TokenAirdrop` transaction - exposes `TokenAirdropTransactionBody` proto
+- `TokenCancel` transaction - exposes `TokenCancelAirdropTransactionBody` proto
+- `TokenClaim` transaction - exposes `TokenClaimAirdropTransactionBody` proto
+- `TokenReject` transaction - exposes `TokenRejectTransactionBody` proto
+- `Token?Flow` that supports rejection and dissociation to make it easier for devs
+
+The SDKs must update the `TransactionRecord` to return the following information
+- `new_pending_airdrops`
+
+### MAPI (Mirror Node API)
+
+- New `/api/v1/accounts/{senderIdOrEvmAddress}/airdrops/outstanding` - List of outstanding airdrops sent by `senderIdOrEvmAddress` which have not been claimed by recipients
+    
+    ```json
+       {
+    			"airdrops": [
+            {
+                "amount": 333,
+                "receiver_id": "0.0.999",
+                "sender_id": "0.0.222",
+    						"serial_number": null,
+                "token_id": "0.0.111"
+            },
+            {
+                "amount": 555,
+                "receiver_id": "0.0.999",
+                "sender_id": "0.0.222",
+    						"serial_number": null,
+                "token_id": "0.0.444"
+            },
+            {
+                "amount": null,
+                "receiver_id": "0.0.999",
+                "sender_id": "0.0.777",
+                "serial_number": 888,
+                "token_id": "0.0.666"
+            }
+    	    ],
+          "links": {
+              "next": null
+          }
+    		}
+    ```
+    
+    - `limit` - The maximum number of airdrops to return in the response. Defaults to `25` with a max of `100`.
+    - `order` - The direction to sort the items in the response. Can be `asc` or `desc` with a default of `asc`.
+    - `receiver.id` - The receiver account the outstanding airdrop was intended for.
+    - `serialnumber` - The specific serial number associated with airdrop. Supports `eq`, `gt`, `gte`, `lt`, and `lte` operators. Only one occurrence is allowed.
+    - `token.id` - The token ID this airdrop is associated with. Supports `eq`, `gt`, `gte`, `lt`, and `lte` operators. Only one occurrence is allowed.
+- New `/api/v1/accounts/{receiverIdOrEvmAddress}/airdrops/pending`) - List of pending airdrops that `receiverIdOrEvmAddress` has not yet claimed.
+    
+    ```json
+       {
+    			"airdrops": [
+            {
+                "amount": 333,
+                "receiver_id": "0.0.999",
```

Comment:
Similar to my question above, if this response should contain entries with the same `receiver_id` isn't that redundant and can we instead omit it from the response body?